plugins/osdn/factory/factory.go

```diff
@@ -24,3 +24,10 @@ func NewPlugin(pluginType string, osClient *osclient.Client, kClient *kclient.Cl
 
 	return nil, nil, nil
 }
+
+func IsMultitenantNetworkPlugin(pluginType string) bool {
```

Comment:
Would it make things simpler to just always do the VNID allocation stuff, even when using ovs-subnet? (The code would just ignore the VNIDs when creating OVS flows in that case.)